src/diffpy/pdfgui/gui/dummyfile.py

```diff
@@ -0,0 +1 @@
+#Hi Simon Billinge. This is Beric's dummy branch. 
```

Comment:
to play with the workflow, can you make edits here?

Reply:
Done.

Reply:
@sbillinge Sorry, forgot to mention you in my reply here. If everything looks good I'll close the PR. 
Thank you!

Reply:
Yes, great job, I think you got it!  Looking forward to working with you!